### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by roidelapluie** [prometheus-community/community#29] [General] [answered]
## 👋 Welcome!
  We’re **evaluating** Discussions, as well as **other options** as a place to connect with other members of our community.you.

Currently the purpose of this is testing only and assessing the functionalities of this solution vs the others solutions out there.

**Comment by Xtigyro** [comment, marked as answer]
Sounds good - thanks, @roidelapluie!

**Comment by roidelapluie** [comment]
test

On 19 Aug 08:41, Miroslav Hadzhiev wrote:
> Sounds good - thanks, @roidelapluie!
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/prometheus-community/community/discussions/29#discussioncomment-53214

-- 
 (o-    Julien Pivotto
 //\    Open-Source Consultant
 V_/_   Inuits - https://www.inuits.eu
